### DISCUSSION TITLE
ScrollArea

### DISCUSSION THREAD

**Discussion by FlatMapIO** [chakra-ui/ark#2150] [Q&A]
ScrollArea is a very important and complex component, does ark-ui have any plans to implement a ScrollArea component like [radix](https://www.radix-ui.com/primitives/docs/components/scroll-area) ?

**Comment by cschroeter** [comment]
There a many sort of components we will add in the future. Make sure to checkout our [roadmap](https://ark-ui.canny.io/feature-requests?search=scroll ) to see what is on the wishlist
